# TurretFeedback.msg
## Turret positional feedback, in radians

time stamp

float32 pitch
float32 yaw
